src/module/actor/vehicle/sheet.ts

```diff
@@ -40,19 +40,18 @@ export class VehicleSheetPF2e extends ActorSheetPF2e<VehiclePF2e> {
         const actorData = sheetData.actor;
 
         // Actions
-        const actions: Record<"action" | "reaction" | "free", { label: string; actions: RawObject<ActionItemData>[] }> =
-            {
-                action: { label: game.i18n.localize("PF2E.ActionsActionsHeader"), actions: [] },
-                reaction: { label: game.i18n.localize("PF2E.ActionsReactionsHeader"), actions: [] },
-                free: { label: game.i18n.localize("PF2E.ActionsFreeActionsHeader"), actions: [] },
-            };
+        const actions: Record<"action" | "reaction" | "free", { label: string; actions: ActionItemPF2e[] }> = {
+            action: { label: game.i18n.localize("PF2E.ActionsActionsHeader"), actions: [] },
+            reaction: { label: game.i18n.localize("PF2E.ActionsReactionsHeader"), actions: [] },
+            free: { label: game.i18n.localize("PF2E.ActionsFreeActionsHeader"), actions: [] },
+        };
 
         // Actions
         for (const item of this.actor.itemTypes.action.sort((a, b) => a.sort - b.sort)) {
-            const itemData = item.toObject(false);
+            const itemData = item.toObject(false) as unknown as ActionItemPF2e;
             const img = getActionIcon(item.actionCost);
             const actionType = item.actionCost?.type ?? "free";
-            actions[actionType].actions.push({ ...itemData, img });
+            actions[actionType].actions.push({ ...itemData, img } as ActionItemPF2e);
```

Comment:
I don't think item.toObject(false) is an ActionItemPF2e :(

Reply:
So you think it's a thing that went away after V9?

Reply:
its more that the type of the map could be anything. Why do an as unknown as ActionItemPF2e rather than using the return type of toObject()?.

Reply:
Because it's also not the source data. The ideal type is closer to the Document than the source

Reply:
So long as these get fixed soonish sure. It just bugs me that I just got this out of any-land and now we're returning right back into it.